aiosparql/client.py

```diff
@@ -166,3 +168,15 @@ def _pretty_print_query(self, query: str) -> str:
                     code=exc.code, message=exc.message, headers=exc.headers,
                     explanation=explanation)
             return await resp.json()
+
+    @asyncio.coroutine
+    def close(self):
+        # NOTE: TypeError: object _CoroGuard can't be used in 'await'
+        #       expression
+        yield from self.session.close()
```

Comment:
This is kinda annoying. I'm not sure yet what's happening and I cannot reproduce it easily with aiohttp alone.